connectors/src/connectors/google_drive/temporal/workflows.ts

```diff
@@ -68,6 +72,25 @@ export async function googleDriveFullSync({
     foldersToBrowse = await getFoldersToSync(connectorId);
   }
 
```

Comment:
IMO, we should always provide an array for `folderToBrowse` and if it's empty it means all folders, if not then only sync the one provided, WDYT?

Reply:
Done